### ISSUE TITLE
Video trimming handle adjustments are not announced to a screen reader

### ISSUE THREAD

**Issue by bmattb** [swissspidy/web-stories-wp#12136]
## Bug Description

When using the self-record functionality and trimming the output of this, the video handle adjustments are not announced to a screen reader

## Additional Context

- Plugin Version:
- WordPress Version:
- Operating System:
- Browser:
